NEWS.md

```diff
@@ -1 +1,7 @@
-## 1.0.0 Unreleased
+## 1.1.0 Unreleased
```

Comment:
shoudn`t this line be: ## 27/05/2021 v1.0.0 - Released?

Reply:
I usually use existing format but it's OK for me to add date.